modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/download/task/DownloadEpisodeTask.kt

```diff
@@ -149,9 +149,7 @@ class DownloadEpisodeTask @AssistedInject constructor(
             }
 
             LogBuffer.i(LogBuffer.TAG_BACKGROUND_TASKS, "Worker Downloading episode ${episode.title} ${episode.uuid}")
-            if (!Util.isWearOs(context)) {
-                setForegroundAsync(createForegroundInfo())
-            }
```

Comment:
`setForegroundAsync(...)` was not allowed for the Wear App in 
https://github.com/Automattic/pocket-casts-android/pull/821#discussion_r1128883000 as it threw an error for notifications on the Wear app but the notification issue was fixed in the following PR https://github.com/Automattic/pocket-casts-android/pull/855 so it should be ok to remove the guarding condition.